channeldb/error.go

```diff
@@ -31,30 +31,6 @@ var (
 	// created.
 	ErrNoPastDeltas = fmt.Errorf("channel has no recorded deltas")
 
-	// ErrInvoiceNotFound is returned when a targeted invoice can't be
```

Comment:
Commit message still references the old pkg name: `lnd/channeldb/types`

Reply:
fixed